src/shared/dom.js

```diff
@@ -22,6 +22,20 @@ export function teardownEach ( iterations, detach, start ) {
 	}
 }
 
+export function spreadAttributes ( node, obj ) {
+	if (obj !== null && typeof obj === 'object') {
+		for ( var property in obj ) {
```

Comment:
Wow I for some reason thought Object.keys was ES6, but it's actually ES5! Definitely will use that.